From 8a1f8c959995b08241a32f30e1a3fd9928efa31d Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <nornagon@nornagon.net>
Date: Thu, 18 Apr 2019 11:07:51 -0700
Subject: Revert "Roll clang 356356:357569."

This reverts commit a50f2f33843d26fd7c0d1f1a2331aa45a392c6cd.

This roll broke ARM builds. Tracking bug: crbug.com/953815

Patch-Filename: revert_roll_clang_356356_357569.patch
---
 tools/clang/scripts/update.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/clang/scripts/update.py b/tools/clang/scripts/update.py
index 145232ac6a27..8bbf3d6f85a8 100755
--- a/tools/clang/scripts/update.py
+++ b/tools/clang/scripts/update.py
@@ -35,7 +35,7 @@ import zipfile
 # Do NOT CHANGE this if you don't know what you're doing -- see
 # https://chromium.googlesource.com/chromium/src/+/master/docs/updating_clang.md
 # Reverting problematic clang rolls is safe, though.
-CLANG_REVISION = '357692'
+CLANG_REVISION = '356356'

 CLANG_REVISION = os.environ.get('LLVM_FORCE_REVISION', CLANG_REVISION).strip()

@@ -45,7 +45,7 @@ if use_head_revision:
   CLANG_REVISION = 'HEAD'

 # This is incremented when pushing a new build of Clang at the same revision.
-CLANG_SUB_REVISION=1
+CLANG_SUB_REVISION=3

 PACKAGE_VERSION = "%s-%s" % (CLANG_REVISION, CLANG_SUB_REVISION)

--
2.19.1.windows.1

